src/main/java/info/mywinecellar/config/WebSecurityConfig.java

```diff
@@ -25,8 +25,11 @@ SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         // Public access to login, landing, and error pages
         http.authorizeRequests().antMatchers("/", "/login", "/errorpage").permitAll();
 
-        // Public access to countries, grapes, wines, wine reviews, tasting notes, and producers
-        http.authorizeRequests().antMatchers("/d/**", "/grape/**", "/review/**", "/tastingnotes/**").permitAll();
+        // Public access to countries, wines, wine reviews, tasting notes, and producers
+        http.authorizeRequests().antMatchers("/d/**", "/review/**", "/tastingnotes/**").permitAll();
+        
```

Comment:
This isn't much of a change here so should go back to the way it was

Reply:
Removing `/grape/**` and adding below with a single wildcard (`/grape/*`) is to differentiate between other full access endpoints (`/d`, `/review`, `/tastingnotes`) and to maintain access to the grape list and detail endpoint but prevent the editing for non-authenticated users. 
Further details are in the following comment.